merge/ignore_test.go

```diff
@@ -174,10 +174,8 @@ func TestIgnoredFieldsUsesVersions(t *testing.T) {
 					APIVersion: "v2",
 				},
 				Apply{
-					Manager: "apply-one",
-					Object: `
-						mapOfMapsRecursive:
-					`,
+					Manager:    "apply-one",
+					Object:     ``,
```

Comment:
hm I can see why you changed this test case but I think we need to know what the output is if clients change nothing?

Reply:
I guess my ask is, leave all the inputs the same and change the outputs in these tests. And copy the cases and modify them if you want to test different inputs.

Reply:
I've made the change you requested. I understand that as a reviewer you want to see how the behavior changed, but I think the test is better the way it was written before.